main.go

```diff
@@ -303,7 +303,10 @@ func main() {
 					if r.TLS != nil {
 						authType := yggdrasilAPIHandler.GetAuthType(r)
 						if !mtls.VerifyRequest(r, authType, opts, CACertChain) {
-							w.WriteHeader(http.StatusUnauthorized)
+							// w.WriteHeader(http.StatusUnauthorized)
+							// return
```

Comment:
Are these comments necessary?

Reply:
When ECOPROJECT task will be in there, I'll uncomment, but I do not want to break the CI too much on device-changes.

Reply:
perhaps worth adding FIXME or TODO comment to make this obvious (and avoid unintentional clean-up PRs by others)

Reply:
Is just in the next line

Reply:
:-) How did I miss it...